packages/aws-otel-collector/aws-otel-collector.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=AWS OTEL collector
+After=network-online.target configured.target
+Wants=network-online.target configured.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/aws-otel-collector --config /etc/aws-otel-collector.yaml
+KillMode=process
+Restart=on-failure
```

Why not always restart? If the process decides it should exit (for whatever reason), we typically want the host to self-heal.

```suggestion
Restart=always
```

---

Yeah, so we expect the collector to always be running (aka never exit cleanly) which is why I left it as `on-failure`.

However, I don't see an issue with changing to `always` just to be safe (also I see that's the general Bottlerocket convention besides for the `wickedd` services), so I can update this.